grakn-test-tools/src/main/java/ai/grakn/util/EmbeddedRedis.java

```diff
@@ -76,19 +74,36 @@ public static void start(int port){
             } else {
                 LOG.warn("Redis already running.");
             }
+        } catch (Exception e) {
+            LOG.warn("Failure to start redis on port {}, maybe running alredy", port, e);
         }
     }
 
     /**
      * Stops the embedded redis
      */
     public static void stop(){
-        if (redisServer != null && REDIS_COUNTER.decrementAndGet() <= 0) {
+        try {
             LOG.info("Stopping Redis...");
             redisServer.stop();
             LOG.info("Redis stopped.");
-        } else {
-            LOG.warn("Called stop while {} redis instances are running", REDIS_COUNTER);
+        } catch (Exception e) {
+            LOG.warn("Failure while stopping redis", e);
+        }
+    }
+
+    public static void forceStart(int redisPort) {
```

Comment:
The fact that this method exists makes me think we should REALLY try to mock out redis properly.